src/app/components/Metadata/index.jsx

```diff
@@ -63,15 +98,15 @@ const Metadata = ({
         />
       ))}
       {renderAmpHtml(ampLink, isAmp)}
-      <meta name="article:author" content={articleAuthor} />
-      <meta name="article:modified_time" content={timeLastPublished} />
-      <meta name="article:published_time" content={timeFirstPublished} />
-      {articleSection ? (
-        <meta name="article:section" content={articleSection} />
-      ) : null}
-      {metaTags.map(tag => (
-        <meta name="article:tag" content={tag} key={tag} />
-      ))}
+      {type === 'article'
```

Comment:
I suggest adding a boolean prop `showArticleTags` to the component, then rendering article tags conditional on that. 
```suggestion
      {showArticleTags ?
```